src/lib/Libpython/shared_python_utils.c

```diff
@@ -43,28 +43,62 @@
 #include "pbs_internal.h"
 #include "log.h"
 
+
+#ifdef WIN32
+
+/**
+ * @brief get_py_homepath
+ * 	Find and return where python home is located
+ *
+ * @param[in] homepath - buffer to copy python home path
+ *
+ * @return int
+ * @retval 0 - Success
+ * @retval 1 - Fail
+ */
```

Comment:
Add to doxygen that the caller is responsible for free()ing the memory allocated